### ISSUE TITLE
🐛 [cloud_firestore] [web] Error: [cloud_firestore/unknown] Expected a value of type 'Object', but got one of type 'Null'

### ISSUE THREAD

**Issue by devj3ns** [FirebaseExtended/flutterfire#5334]
## Bug report

**Describe the bug**

When trying to update a document with a null value I get this error:
`Error: [cloud_firestore/unknown] Expected a value of type 'Object', but got one of type 'Null'`

When I change the null's to `''` it works and I don't get this error. I already tried to change ` Object?` to `dynamic`, which makes no difference.

**My code**

```
Map<String, Object?> toDocument() {
    return {
      'title': null,
      'text': null,
    };
  }
```
```
await _messageCollection.doc(docId).update(message.toDocument());
```
---

### Stacktrace
<details><summary>Click To Expand</summary>

```
Error: [cloud_firestore/unknown] Expected a value of type 'Object', but got one of type 'Null'
    at Object.throw_ [as throw] (http://localhost:63165/dart_sdk.js:5032:11)
    at document_reference_web.DocumentReferenceWeb.new.update (http://localhost:63165/packages/cloud_firestore_web/src/write_batch_web.dart.lib.js:713:23)
    at update.next (<anonymous>)
    at runBody (http://localhost:63165/dart_sdk.js:37229:34)
    at Object._async [as async] (http://localhost:63165/dart_sdk.js:37260:7)
    at document_reference_web.DocumentReferenceWeb.new.update (http://localhost:63165/packages/cloud_firestore_web/src/write_batch_web.dart.lib.js:707:20)
    at cloud_firestore.DocumentReference.__.update (http://localhost:63165/packages/cloud_firestore/cloud_firestore.dart.lib.js:636:32)
    at feed_repository.FeedRepository.new.updateMessage (http://localhost:63165/packages/feed_repository/src/feed_repository.dart.lib.js:217:72)
    at updateMessage.next (<anonymous>)
    at runBody (http://localhost:63165/dart_sdk.js:37229:34)
    at Object._async [as async] (http://localhost:63165/dart_sdk.js:37260:7)
    at feed_repository.FeedRepository.new.updateMessage (http://localhost:63165/packages/feed_repository/src/feed_repository.dart.lib.js:206:20)
    at add_edit_message_cubit.AddEditMessageCubit.new.onFormSubmitted (http://localhost:63165/packages/nestwaerme_kita_app/feed/add_edit_message_page/cubit/add_edit_message_cubit.dart.lib.js:233:51)
    at onFormSubmitted.next (<anonymous>)
    at runBody (http://localhost:63165/dart_sdk.js:37229:34)
    at Object._async [as async] (http://localhost:63165/dart_sdk.js:37260:7)
    at add_edit_message_cubit.AddEditMessageCubit.new.onFormSubmitted (http://localhost:63165/packages/nestwaerme_kita_app/feed/add_edit_message_page/cubit/add_edit_message_cubit.dart.lib.js:228:20)
    at submitForm (http://localhost:63165/packages/nestwaerme_kita_app/feed/add_edit_message_page/widgets/add_edit_message_form.dart.lib.js:1051:93)
    at ink_well._InkResponseState.new.[_handleTap] (http://localhost:63165/packages/flutter/src/material/icon_button.dart.lib.js:50084:42)
    at tap.TapGestureRecognizer.new.invokeCallback (http://localhost:63165/packages/flutter/src/gestures/recognizer.dart.lib.js:201:18)
    at tap.TapGestureRecognizer.new.handleTapUp (http://localhost:63165/packages/flutter/src/gestures/tap.dart.lib.js:417:40)
    at tap.TapGestureRecognizer.new.[_checkUp] (http://localhost:63165/packages/flutter/src/gestures/tap.dart.lib.js:223:12)
    at tap.TapGestureRecognizer.new.handlePrimaryPointer (http://localhost:63165/packages/flutter/src/gestures/tap.dart.lib.js:169:23)
    at tap.TapGestureRecognizer.new.handleEvent (http://localhost:63165/packages/flutter/src/gestures/recognizer.dart.lib.js:456:16)
    at pointer_router.PointerRouter.new.[_dispatch] (http://localhost:63165/packages/flutter/src/gestures/pointer_router.dart.lib.js:103:9)
    at http://localhost:63165/packages/flutter/src/gestures/pointer_router.dart.lib.js:138:26
    at LinkedMap.new.forEach (http://localhost:63165/dart_sdk.js:25048:11)
    at pointer_router.PointerRouter.new.[_dispatchEventToRoutes] (http://localhost:63165/packages/flutter/src/gestures/pointer_router.dart.lib.js:135:29)
    at pointer_router.PointerRouter.new.route (http://localhost:63165/packages/flutter/src/gestures/pointer_router.dart.lib.js:127:37)
    at binding$5.WidgetsFlutterBinding.new.handleEvent (http://localhost:63165/packages/flutter/src/gestures/binding.dart.lib.js:330:26)
    at binding$5.WidgetsFlutterBinding.new.dispatchEvent (http://localhost:63165/packages/flutter/src/gestures/binding.dart.lib.js:313:24)
    at binding$5.WidgetsFlutterBinding.new.dispatchEvent (http://localhost:63165/packages/flutter/src/rendering/layer.dart.lib.js:6071:13)
    at binding$5.WidgetsFlutterBinding.new.[_handlePointerEventImmediately] (http://localhost:63165/packages/flutter/src/gestures/binding.dart.lib.js:284:14)
    at binding$5.WidgetsFlutterBinding.new.handlePointerEvent (http://localhost:63165/packages/flutter/src/gestures/binding.dart.lib.js:257:43)
    at binding$5.WidgetsFlutterBinding.new.[_flushPointerEventQueue] (http://localhost:63165/packages/flutter/src/gestures/binding.dart.lib.js:246:14)
    at binding$5.WidgetsFlutterBinding.new.[_handlePointerDataPacket] (http://localhost:63165/packages/flutter/src/gestures/binding.dart.lib.js:236:65)
    at Object.invoke1 (http://localhost:63165/dart_sdk.js:173249:7)
    at _engine.EnginePlatformDispatcher.__.invokeOnPointerDataPacket (http://localhost:63165/dart_sdk.js:155201:15)
    at _engine.PointerBinding.__.[_onPointerData] (http://localhost:63165/dart_sdk.js:155815:49)
    at http://localhost:63165/dart_sdk.js:156224:26
    at http://localhost:63165/dart_sdk.js:156183:16
    at http://localhost:63165/dart_sdk.js:155908:11

```

</details>

---

### Flutter doctor

Run `flutter doctor` and paste the output below:

<details><summary>Click To Expand</summary>

```
[✓] Flutter (Channel stable, 2.0.1, on Microsoft Windows [Version 10.0.19042.867], locale de-DE)
[✓] Android toolchain - develop for Android devices (Android SDK version 29.0.3)
[✓] Chrome - develop for the web
[✓] Android Studio (version 4.1.0)
[✓] VS Code (version 1.51.1)
[✓] Connected device (2 available)
```

</details>

---

### Flutter dependencies

Run `flutter pub deps -- --style=compact` and paste the output below:

<details><summary>Click To Expand</summary>

```
Dart SDK 2.12.0
Flutter SDK 2.0.1
nestwaerme_kita_app 0.9.9+9

dependencies:
- absence_repository 2.0.0 [flutter firebase_core cloud_firestore equatable csv intl extensions]
- appointment_repository 2.0.0 [flutter firebase_core cloud_firestore equatable extensions]
- cached_network_image 3.0.0-nullsafety [flutter flutter_cache_manager octo_image]
- datetime_picker_formfield 2.0.0 [flutter intl]
- extensions 2.0.0
- feed_repository 2.0.0 [flutter firebase_core cloud_firestore firebase_storage equatable extensions]
- file_picker 3.0.0 [flutter flutter_web_plugins flutter_plugin_android_lifecycle plugin_platform_interface]
- firebase_crashlytics 1.0.0 [firebase_core firebase_core_platform_interface firebase_crashlytics_platform_interface flutter stack_trace]
- flash 1.5.0 [flutter]
- flutter 0.0.0 [characters collection meta typed_data vector_math sky_engine]
- flutter_bloc 7.0.0-nullsafety.4 [flutter bloc provider]
- flutter_hooks 0.16.0 [flutter]
- flutter_localizations 0.0.0 [flutter intl characters clock collection meta path typed_data vector_math]
- font_awesome_flutter 9.0.0-nullsafety [flutter]
- google_fonts 2.0.0 [flutter http path_provider crypto]
- google_nav_bar 5.0.0 [flutter]
- intl 0.17.0 [clock path]
- notification_repository 2.0.0 [flutter firebase_core firebase_messaging http]
- pdf_flutter 1.2.0 [flutter flutter_cache_manager http path_provider]
- url_launcher 6.0.2 [flutter url_launcher_platform_interface url_launcher_linux url_launcher_macos url_launcher_windows url_launcher_web]
- url_strategy 0.2.0 [flutter_web_plugins]
- user_repository 2.0.0 [flutter firebase_core firebase_auth firebase_storage cloud_firestore cloud_functions equatable flutter_secure_storage extensions]

dev dependencies:
- flutter_launcher_icons 0.9.0 [args image path yaml]
- flutter_test 0.0.0 [flutter test_api path fake_async clock stack_trace vector_math async boolean_selector characters charcode collection matcher meta source_span stream_channel string_scanner term_glyph typed_data]
- pedantic 1.11.0

transitive dependencies:
- archive 3.1.2 [crypto path]
- args 2.0.0
- async 2.5.0 [collection]
- bloc 7.0.0-nullsafety.3 [meta]
- boolean_selector 2.1.0 [source_span string_scanner]
- characters 1.1.0
- charcode 1.2.0
- clock 1.1.0
- cloud_firestore 1.0.1 [cloud_firestore_platform_interface cloud_firestore_web firebase_core firebase_core_platform_interface flutter meta]
- cloud_firestore_platform_interface 4.0.0 [collection firebase_core flutter meta plugin_platform_interface]
- cloud_firestore_web 1.0.1 [cloud_firestore_platform_interface firebase_core firebase_core_web flutter flutter_web_plugins js]
- cloud_functions 1.0.0 [cloud_functions_platform_interface cloud_functions_web firebase_core firebase_core_platform_interface flutter]
- cloud_functions_platform_interface 5.0.0 [firebase_core flutter meta plugin_platform_interface]
- cloud_functions_web 4.0.0 [cloud_functions_platform_interface firebase_core firebase_core_web flutter flutter_web_plugins js]
- collection 1.15.0
- crypto 3.0.0 [collection typed_data]
- csv 5.0.0-nullsafety.0
- equatable 2.0.0 [collection meta]
- fake_async 1.2.0 [clock collection]
- ffi 1.0.0
- file 6.1.0 [meta path]
- firebase_auth 1.0.1 [firebase_auth_platform_interface firebase_auth_web firebase_core firebase_core_platform_interface flutter meta]
- firebase_auth_platform_interface 4.0.0 [firebase_core flutter meta plugin_platform_interface]
- firebase_auth_web 1.0.2 [firebase_auth_platform_interface firebase_core firebase_core_web flutter flutter_web_plugins http_parser intl js meta]
- firebase_core 1.0.1 [firebase_core_platform_interface firebase_core_web flutter meta]
- firebase_core_platform_interface 4.0.0 [flutter meta plugin_platform_interface]
- firebase_core_web 1.0.1 [firebase_core_platform_interface flutter flutter_web_plugins js meta]
- firebase_crashlytics_platform_interface 2.0.0 [collection firebase_core flutter meta plugin_platform_interface]
- firebase_messaging 9.0.0 [firebase_core firebase_core_platform_interface firebase_messaging_platform_interface firebase_messaging_web flutter meta]
- firebase_messaging_platform_interface 2.0.0 [firebase_core flutter meta plugin_platform_interface]
- firebase_messaging_web 1.0.0 [firebase_core firebase_core_web firebase_messaging_platform_interface flutter flutter_web_plugins js meta]
- firebase_storage 8.0.0 [firebase_core firebase_core_platform_interface firebase_storage_platform_interface firebase_storage_web flutter]
- firebase_storage_platform_interface 2.0.0 [collection firebase_core flutter meta plugin_platform_interface]
- firebase_storage_web 1.0.0 [async firebase_core firebase_core_web firebase_storage_platform_interface flutter flutter_web_plugins http js meta]
- flutter_blurhash 0.5.4-nullsafety.0 [flutter meta pedantic]
- flutter_cache_manager 3.0.0-nullsafety.1 [clock collection file flutter http image path path_provider pedantic rxdart sqflite uuid]
- flutter_plugin_android_lifecycle 2.0.0 [flutter]
- flutter_secure_storage 4.1.0 [meta flutter]
- flutter_web_plugins 0.0.0 [flutter js characters collection meta typed_data vector_math]
- http 0.13.0 [http_parser meta path pedantic]
- http_parser 4.0.0 [charcode collection source_span string_scanner typed_data]
- image 3.0.1 [archive meta xml]
- js 0.6.3
- matcher 0.12.10 [stack_trace]
- meta 1.3.0
- nested 1.0.0 [flutter]
- octo_image 1.0.0-nullsafety.1 [flutter flutter_blurhash]
- path 1.8.0
- path_provider 2.0.1 [flutter path_provider_platform_interface path_provider_macos path_provider_linux path_provider_windows]
- path_provider_linux 2.0.0 [path xdg_directories path_provider_platform_interface flutter]
- path_provider_macos 2.0.0 [flutter]
- path_provider_platform_interface 2.0.1 [flutter meta platform plugin_platform_interface]
- path_provider_windows 2.0.0 [path_provider_platform_interface meta path flutter ffi win32]
- petitparser 4.0.2 [meta]
- platform 3.0.0
- plugin_platform_interface 2.0.0 [meta]
- process 4.1.0 [file path platform]
- provider 5.0.0 [collection flutter nested]
- rxdart 0.26.0
- sky_engine 0.0.99
- source_span 1.8.0 [charcode collection path term_glyph]
- sqflite 2.0.0+2 [flutter sqflite_common path]
- sqflite_common 2.0.0+1 [synchronized path meta]
- stack_trace 1.10.0 [path]
- stream_channel 2.1.0 [async]
- string_scanner 1.1.0 [charcode source_span]
- synchronized 3.0.0
- term_glyph 1.2.0
- test_api 0.2.19 [async boolean_selector collection meta path source_span stack_trace stream_channel string_scanner term_glyph matcher]
- typed_data 1.3.0 [collection]
- url_launcher_linux 2.0.0 [flutter]
- url_launcher_macos 2.0.0 [flutter]
- url_launcher_platform_interface 2.0.2 [flutter plugin_platform_interface]
- url_launcher_web 2.0.0 [url_launcher_platform_interface meta flutter flutter_web_plugins]
- url_launcher_windows 2.0.0 [flutter]
- uuid 3.0.1 [crypto]
- vector_math 2.1.0
- win32 2.0.0 [ffi]
- xdg_directories 0.2.0 [meta path process]
- xml 5.0.2 [collection meta petitparser]
- yaml 3.1.0 [collection source_span string_scanner]
```

</details>

---

**Comment by rrousselGit** [comment]
Do you have a stacktrace for the error?

**Comment by devj3ns** [comment]
Oh, yes. Sorry. I forgot to add that to the issue. Now it's added.

**Comment by rrousselGit** [comment]
Hum, I haven't been able to reproduce it locally, nor do I see anything asking for an `Object` in the stacktrace.

Could you share a minimalist example to reproduce the issue?

**Comment by devj3ns** [comment]
Hmm, strange. With what firebase.js version have you tested?  I am using 7.20.0 in my index.html

**Comment by devj3ns** [comment]
This works:
```
final Map<String, Object?> messageData = {
      'title': '',
      'text': '',
    };

    await FirebaseFirestore.instance
        .collection('users/${feed.id}/messages')
        .add(messageData);
```
This not:
```
final Map<String, Object?> messageData = {
      'title': null,
      'text': null,
    };

    await FirebaseFirestore.instance
        .collection('users/${feed.id}/messages')
        .add(messageData);
```

**Comment by devj3ns** [comment]
I just realized that an error I got yesterday says the same as this one. See: https://github.com/flutter/flutter/issues/78017

I think something is wrong with my project (with the web version), but I don't know what.

**Comment by devj3ns** [comment]
When I run my web app with `--no-sound-null-safety` everything works. 

But all my code and dependencies are migrated. I don't understand why I get this error.

**Comment by markusaksli-nc** [comment]
I'm seeing this as well with
```yaml
  firebase_core: ^1.0.1
  cloud_firestore: ^1.0.1
```
```html
  <script src="https://www.gstatic.com/firebasejs/7.20.0/firebase-app.js"></script>
  <script src="https://www.gstatic.com/firebasejs/7.20.0/firebase-firestore.js"></script>
```
<details>
 <summary>flutter doctor -v</summary>

```console
[√] Flutter (Channel master, 2.1.0-13.0.pre.86, on Microsoft Windows [Version 10.0.19041.867], locale et-EE)
    • Flutter version 2.1.0-13.0.pre.86 at C:\Development\flutter_master
    • Framework revision 40e3489254 (2 days ago), 2021-03-13 17:43:03 -0500
    • Engine revision 9ec46417bd
    • Dart version 2.13.0 (build 2.13.0-134.0.dev)

[√] Android toolchain - develop for Android devices (Android SDK version 30.0.2)
    • Android SDK at C:\Users\marku\AppData\Local\Android\sdk
    • Platform android-30, build-tools 30.0.2
    • Java binary at: C:\Users\marku\AppData\Local\JetBrains\Toolbox\apps\AndroidStudio\ch-0\201.7042882\jre\bin\java
    • Java version OpenJDK Runtime Environment (build 1.8.0_242-release-1644-b01)
    • All Android licenses accepted.

[√] Chrome - develop for the web
    • Chrome at C:\Program Files\Google\Chrome\Application\chrome.exe

[√] Visual Studio - develop for Windows (Visual Studio Community 2019 16.7.7)
    • Visual Studio at C:\Program Files (x86)\Microsoft Visual Studio\2019\Community
    • Visual Studio Community 2019 version 16.7.30621.155
    • Windows 10 SDK version 10.0.18362.0

[√] Android Studio (version 4.1.0)
    • Android Studio at C:\Users\marku\AppData\Local\JetBrains\Toolbox\apps\AndroidStudio\ch-0\201.7042882
    • Flutter plugin version 54.0.1
    • Dart plugin version 201.9335
    • Java version OpenJDK Runtime Environment (build 1.8.0_242-release-1644-b01)

[√] IntelliJ IDEA Ultimate Edition (version 2020.3)
    • IntelliJ at C:\Users\marku\AppData\Local\JetBrains\Toolbox\apps\IDEA-U\ch-0\203.7148.57
    • Flutter plugin can be installed from:
       https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
       https://plugins.jetbrains.com/plugin/6351-dart

[√] Connected device (4 available)
    • sdk gphone x86 arm (mobile) • emulator-5554 • android-x86    • Android 11 (API 30) (emulator)
    • Windows (desktop)           • windows       • windows-x64    • Microsoft Windows [Version 10.0.19041.867]
    • Chrome (web)                • chrome        • web-javascript • Google Chrome 89.0.4389.82
    • Edge (web)                  • edge          • web-javascript • Microsoft Edge 89.0.774.50

• No issues found!
```
</details>
<details>
 <summary>Minimal reproducible code sample</summary>

```dart
import 'package:cloud_firestore/cloud_firestore.dart';
import 'package:firebase_core/firebase_core.dart';
import 'package:flutter/material.dart';

Future<void> main() async {
  WidgetsFlutterBinding.ensureInitialized();
  await Firebase.initializeApp();
  runApp(MaterialApp(
    title: "Sample",
    home: MyApp(),
  ));
}

class MyApp extends StatelessWidget {
  @override
  Widget build(BuildContext context) {
    return Scaffold(
      appBar: AppBar(title: const Text("Sample"),),
      body: Center(
        child: ElevatedButton(
          child: const Text("Throw"),
          onPressed: () async {
            final Map<String, Object?> messageData = {
              'title': null,
              'text': null,
            };
            await FirebaseFirestore.instance.collection('test').doc('nulls').set(messageData);
          },
        ),
      ),
    );
  }
}
```
</details>
<details>
 <summary>Logs</summary>

```console
Error: [cloud_firestore/unknown] Expected a value of type 'Object', but got one of type 'Null'
    at Object.throw_ [as throw] (http://localhost:62240/dart_sdk.js:5033:11)
    at document_reference_web.DocumentReferenceWeb.new.set (http://localhost:62240/packages/cloud_firestore_web/src/write_batch_web.dart.lib.js:615:23)
    at set.next (<anonymous>)
    at runBody (http://localhost:62240/dart_sdk.js:37375:34)
    at Object._async [as async] (http://localhost:62240/dart_sdk.js:37406:7)
    at document_reference_web.DocumentReferenceWeb.new.set (http://localhost:62240/packages/cloud_firestore_web/src/write_batch_web.dart.lib.js:609:20)
    at cloud_firestore.DocumentReference.__.set (http://localhost:62240/packages/cloud_firestore/cloud_firestore.dart.lib.js:579:32)
    at http://localhost:62240/packages/triage/main.dart.lib.js:306:96
    at Generator.next (<anonymous>)
    at runBody (http://localhost:62240/dart_sdk.js:37375:34)
    at Object._async [as async] (http://localhost:62240/dart_sdk.js:37406:7)
    at http://localhost:62240/packages/triage/main.dart.lib.js:303:280
    at ink_well._InkResponseState.new.[_handleTap] (http://localhost:62240/packages/flutter/src/material/icon_button.dart.lib.js:48447:42)
    at tap.TapGestureRecognizer.new.invokeCallback (http://localhost:62240/packages/flutter/src/gestures/recognizer.dart.lib.js:183:18)
    at tap.TapGestureRecognizer.new.handleTapUp (http://localhost:62240/packages/flutter/src/gestures/tap.dart.lib.js:393:40)
    at tap.TapGestureRecognizer.new.[_checkUp] (http://localhost:62240/packages/flutter/src/gestures/tap.dart.lib.js:205:12)
    at tap.TapGestureRecognizer.new.handlePrimaryPointer (http://localhost:62240/packages/flutter/src/gestures/tap.dart.lib.js:154:23)
    at tap.TapGestureRecognizer.new.handleEvent (http://localhost:62240/packages/flutter/src/gestures/recognizer.dart.lib.js:423:16)
    at pointer_router.PointerRouter.new.[_dispatch] (http://localhost:62240/packages/flutter/src/gestures/pointer_router.dart.lib.js:85:9)
    at http://localhost:62240/packages/flutter/src/gestures/pointer_router.dart.lib.js:115:26
    at LinkedMap.new.forEach (http://localhost:62240/dart_sdk.js:25203:11)
    at pointer_router.PointerRouter.new.[_dispatchEventToRoutes] (http://localhost:62240/packages/flutter/src/gestures/pointer_router.dart.lib.js:113:29)
    at pointer_router.PointerRouter.new.route (http://localhost:62240/packages/flutter/src/gestures/pointer_router.dart.lib.js:108:37)
    at binding$5.WidgetsFlutterBinding.new.handleEvent (http://localhost:62240/packages/flutter/src/gestures/binding.dart.lib.js:362:26)
    at binding$5.WidgetsFlutterBinding.new.dispatchEvent (http://localhost:62240/packages/flutter/src/gestures/binding.dart.lib.js:347:24)
    at binding$5.WidgetsFlutterBinding.new.dispatchEvent (http://localhost:62240/packages/flutter/src/rendering/layer.dart.lib.js:5022:13)
    at binding$5.WidgetsFlutterBinding.new.[_handlePointerEventImmediately] (http://localhost:62240/packages/flutter/src/gestures/binding.dart.lib.js:321:14)
    at binding$5.WidgetsFlutterBinding.new.handlePointerEvent (http://localhost:62240/packages/flutter/src/gestures/binding.dart.lib.js:295:43)
    at binding$5.WidgetsFlutterBinding.new.[_flushPointerEventQueue] (http://localhost:62240/packages/flutter/src/gestures/binding.dart.lib.js:285:14)
    at binding$5.WidgetsFlutterBinding.new.[_handlePointerDataPacket] (http://localhost:62240/packages/flutter/src/gestures/binding.dart.lib.js:276:54)
    at Object.invoke1 (http://localhost:62240/dart_sdk.js:174769:7)
    at _engine.EnginePlatformDispatcher.__.invokeOnPointerDataPacket (http://localhost:62240/dart_sdk.js:156569:15)
    at _engine.PointerBinding.__.[_onPointerData] (http://localhost:62240/dart_sdk.js:157207:49)
    at http://localhost:62240/dart_sdk.js:157625:28
    at http://localhost:62240/dart_sdk.js:157581:16
    at http://localhost:62240/dart_sdk.js:157300:11
```
</details>

**Comment by devj3ns** [comment]
Thanks for testing @markusaksli-nc 

At least I am not the only one with this issue 😂

Do you also have this issue https://github.com/flutter/flutter/issues/78017?

**Comment by markusaksli-nc** [comment]
I tried to read into the stack trace but I didn't exactly understand where this was getting thrown so can't help with that

**Comment by devj3ns** [comment]
> I tried to read into the stack trace but I didn't exactly understand where this was getting thrown so can't help with that

Okay, no problem. It's only a warning, this issue is more important to be fixed.

**Comment by rmill777** [comment]
This is a problem for me as well. I can no longer set fields to null... broke my app trying to upgrade to null safety...

**Comment by rrousselGit** [comment]
fixed by https://github.com/FirebaseExtended/flutterfire/issues/5334

**Comment by devj3ns** [comment]
> fixed by #5334

Hey, you linked the issue itself ;) 
Could you please link the PR which closed the issue?
